lib/saml/url_service.rb

```diff
@@ -92,6 +92,18 @@ def idme_url
       build_sso_url(build_authn_context(LOA::IDME_LOA1_VETS))
     end
 
+    def signup_url
+      @type = 'signup'
+      @query_params[:op] = 'signup'
```

Comment:
The signup_url and idme_signup_url methods look identical. Can they be combined somehow?

Reply:
They are, the idme one is replacing the original but we need to preserve both until the frontend actually switches over, then remove `signup_url`.

Reply:
Ah ok, sounds good1